README.md

```diff
@@ -77,6 +77,23 @@ Each entry in the `servers` section represents a separate virtual host.
     if a string is given, it is interpreted to be `upstream.url`, with `upstream.location` set to `/`,
     and `upstream.name` `$server_name-upstream`
 
+### Setting nginx options directly
+
+To add or override a setting directly in the generated `nginx.conf` file, add desired settings in the `raw_options`
+array. For settings in the `http` section, use a top-level array. For a specific server section, put the array in
+the sections's entry.
+
+```yaml
+raw_options:
+  - keepalive_timeout 10
+servers:
+  webapp:
+    server_name: my-webapp.example.com
+    upstream: uwsgi://webapp:12345
+    raw_options:
```

Comment:
We have two fields called `raw_options`. Reading the code I got lost which part refers to what. I'd suggest changing them to something a bit more verbose, like `http_raw_options` and `server_raw_options`. Or something similar.